dts/bindings/serial/nanoxplore,ngultra-uart.yaml

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2022, NanoXplore
+# SPDX-License-Identifier: Apache-2.0
+
+description: NanoXplore UART
+
+compatible: "nanoxplore,ngultra-uart"
+
+include: uart-controller.yaml
+
+properties:
+    reg:
+      required: true
+
+    current-speed:
+     type: int
+     required: true
+     description: Configure the UART baudrate
```

Comment:
Since the `current-speed` property is defined in `uart-controller.yaml` you only need a `required:true` here like how it is for the `reg` property.

Reply:
Done